packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -0,0 +1,35 @@
+import { html, when } from "@microsoft/fast-element";
+import { Avatar } from "./avatar";
+
+/**
+ * The template for {@link @microsoft/fast-foundation#Avatar} component.
+ * @public
+ */
+export const AvatarTemplate = html<Avatar>`
+    <template>
```

Comment:
should the alt text or some aria stuff be on the template/host element here for the screen reader?

Reply:
All this should be delegated to the image and anchor elements.